workitem/workitem_repository.go

```diff
@@ -158,6 +158,7 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, typeID string, fiel
 		var err error
 		wi.Fields[fieldName], err = fieldDef.ConvertToModel(fieldName, fieldValue)
 		if err != nil {
+			log.Printf("Failed to convert field '%s' to model: %v\r\n", fieldName, err)
```

Comment:
ok, I removed the `log.Printf` statement. The method returns an `errors.NewBadParameterError(fieldName, fieldValue)` anyway.